canonical/store-components
# Enable setting for show more categories

## Done
Added an option to show all categories whilst keeping the "Show more" functionality

## QA
- Go to https://store-components-123.demos.haus/?path=/story/filters--with-show-more-enabled
- Check that all categories are displayed and the "Show less/more" button still works
